doc/introduction/measurements.rst

```diff
@@ -300,6 +300,37 @@ The decorator syntax applies equally well:
     def qnode(x, y):
         (...)
 
+The one-shot transform
+**********************
+
+Devices supporting mid-circuit measurements (defined using
+:func:`~.pennylane.measure`) and conditional operations (defined using
+:func:`~.pennylane.cond`) natively can estimate dynamic circuits by executing
+them one shot at the time. This is the default behaviour of a QNode that has a
+device supporting mid-circuit measurements, as well as any QNode with the
```

Comment:
This is only when in finite-shot mode right? And we're using deferred for analytic still?